amora/models.py

```diff
@@ -242,35 +241,20 @@ def _is_amora_model(candidate: ModuleType) -> bool:
     )
 
 
+@ensure_path
 def amora_model_for_path(path: Path) -> Model:
-    spec = spec_from_file_location(".".join(["amoramodel", path.stem]), path)
-    if spec is None:
-        raise ValueError(f"Invalid path `{path}`. Not a valid Python file.")
-
-    module = module_from_spec(spec)
-
-    if spec.loader is None:
-        raise ValueError(f"Invalid AmoraModel path `{path}`. Unable to load module.")
-
     try:
-        spec.loader.exec_module(module)
-    except ImportError as e:
-        raise ValueError(
-            f"Invalid AmoraModel path `{path}`. Unable to load module."
-        ) from e
+        module = importlib.import_module(path.stem, settings.models_path.name)
+    except ModuleNotFoundError:
+        module = None
```

Comment:
Acredito que a execução da função deveria terminar aqui. O getmembers de None vai retornar `[]`, e vai pro ValueError


```suggestion
    except ModuleNotFoundError as e:
        raise ValueError(f"Invalid path `{path}`") from e
```

Reply:
Eu tinha feito assim inicialmente, mas ai fica duplicado esse `raise ValueError ...`

Reply:
Não acho que DRY se aplique aqui. É uma repetição que vale a pena, já que faz com que eu não precise executar o resto do código na minha cabeça pra saber o que vai acontecer.